File: 2965. Find Missing and Repeated Values.cpp
class Solution {
    public:
        vector<int> findMissingAndRepeatedValues(vector<vector<int>>& grid) 
        {
            map<int,int>mp;
            int n = grid.size();
            for(int i=0;i<n;i++)
            {
                for(int j=0;j<n;j++)
                    mp[grid[i][j]]++;
            }
    
            int rep, miss = 0;
            n = n*n;
            for(int i=1;i<=n;i++)
            {
                if(mp[i]==2) rep = i;
                else if(mp[i]==0) miss = i;
            }
            return {rep,miss};
        }
    };

File: 2965. Find Missing and Repeated Values.py
class Solution:
    def findMissingAndRepeatedValues(self, grid: List[List[int]]) -> List[int]:
        ans = [0, 0]
        store = defaultdict(int)
        for row in grid:
            for val in row:
                store[val] += 1

        val = len(grid) * len(grid)
        while val:
            if store[val] == 0:
                ans[1] = val
            elif store[val] == 2:
                ans[0] = val
            val -= 1
        return ans